lib/Command/Diagnostics.php

```diff
@@ -47,6 +47,18 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 					if ($stat === 'last_seen') {
 						$output->write('last_seen=' . (new \DateTime('@' . $value))->format('Y-m-d H:i:s'));
 					}
+					if ($stat === 'last_started') {
+						$output->write('last_started=' . (new \DateTime('@' . $value))->format('Y-m-d H:i:s'));
+					}
+					if ($stat === 'last_ended') {
+						$output->write('last_ended=' . (new \DateTime('@' . $value))->format('Y-m-d H:i:s'));
+					}
+					if ($stat === 'started_count') {
+						$output->write('started_count=' . $value);
+					}
+					if ($stat === 'last_triggered') {
+						$output->write('last_triggered=' . (new \DateTime('@' . $value))->format('Y-m-d H:i:s'));
+					}
 				}
 				$output->writeln('');
```

Comment:
`writeln` can go inside the loop
also, we can probably use `elseif`s to prevent some cpu cycles since each stat can only be one of the types.

Reply:
ah, I'm missing spaces :D